spec/features/contestant_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe 'Contestants', type: :feature do
+  describe 'Index' do
+    it 'displays a link to challenges, contestant_projects, contestants, and projects' do
+      visit contestants_path
+
+      expect(page).to have_link('Challenges')
+      expect(page).to have_link('Contestants')
+      expect(page).to have_link('Projects')
```

Comment:
Instead of repeating this test, you could also make an `application_spec.rb` feature test and test it once there.